src/main/java/frc/robot/Constants/Constants.java

```diff
@@ -64,16 +63,6 @@ public static class LEDConstants {
 
 	
 	public static class PigeonConstants {
-		public static final int pigeonID = 23;
-	}
-
-	public static class SwervePIDControllerConstants {
-		public static final double driveP = 1.5;
-		public static final double driveI = 0;
-		public static final double driveD = 0;
-
-		public static final double turnP = 2.5;
-		public static final double turnI = 0;
-		public static final double turnD = 0;
+		public static final int pigeonID = 30;
```

Comment:
This is set by YAGSL config. add TODO to remove pigeon instance and use the YAGSL one.